src/main/java/racingcar/model/Car.java

```diff
@@ -0,0 +1,11 @@
+package racingcar.model;
+
+public class Car {
+    int carNumber;
+    public String currentCarMovingStatus;
```

Comment:
현재 움직임을 문자열로 나타내고 있네요.
만약 요구사항이 변경되어 움직인 거리를 이미지로 표현해야 한다면,
지금 구조에서는 많은 수정을 해야 할 수 있습니다. 

얼마나 움직였는지 보여주는 책임을 출력 역할을 가진 ResultView에 위임하는 것이 적절할 것 같아요.
```suggestion
    private int 움직인거리;
```
이렇게 하면 출력 방식이 변경되더라도 대응이 가능합니다.

Reply:
필드를 public으로 노출시키면, 객체가 스스로 판단하여 상태를 변경하는 것이 아니라
외부에서 직접 값을 변경할 수 있어 문제가 발생할 수 있습니다. 
가능하면 필드를 private으로 선언하여 감추고, 값 변경이 필요할 때는 메소드를 통해 제어하는 것이 좋을 것 같습니다.

메소드에 네이밍까지 잘 되었다면 객체가 어떻게 변경되는지도 쉽게 추측할 수 있게되겠네요 😄